files/rsyslog/rsyslog.sh

```diff
@@ -45,15 +45,15 @@ if [ "${CDM_USE_UNDEFINED,,}" = "false" -a "${CDM_KEEP_EMPTY_FIELDS}" = "" -a "$
         if [ "${USE_MMEXTERNAL:-}" = "" ] ; then
             export SKIP_EMPTY=false
             export USE_MMEXTERNAL=true
-            echo "INFO: SKIP_EMPTY was set to false, but USE_MMEXTERNAL was not set and none of the UNDEFINED fields were configured. Enabling USE_MMEXTERNAL. Consider using SKIP_EMPTY=true instead."
+            echo "INFO: SKIP_EMPTY was set to false, but USE_MMEXTERNAL was not set and none of the environment variables to manage undefined and empty fields were configured. Enabling USE_MMEXTERNAL. Consider using SKIP_EMPTY=true instead."
```

Comment:
_USE_MMEXTERNAL was not set _
This means that U`USE_MMEXTERAL` is `true` by default, right? 
_INFO: SKIP_EMPTY was set to false, USE_MMEXTERNAL is true by default but none of the environment variables to manage undefined and empty fields were configured. Consider configuring the CDM_* environment variables _ or consider using SKIP_EMPTY=true instead._

Something to tell the user what to do in this case?